ocaml/xapi/xapi.ml

```diff
@@ -806,7 +806,6 @@ let server_init() =
           "Registering SMAPIv1 plugins", [Startup.OnlyMaster], Sm.register;
           "Starting SMAPIv1 proxies", [Startup.OnlyMaster], Storage_access.start_smapiv1_servers;
           "Initialising SM state", [], Storage_impl.initialise;
-          "Starting SM internal event service", [], Scheduler.start;
```

Comment:
Yeah, sorry, I thought it better to keep the patches apart without squashing because they made reasonable semantic sense even though later ones reversed changes in earlier ones.